packages/RatPack.package/RatPackMiddleware.class/class/page2.st

```diff
@@ -0,0 +1,26 @@
+pages
+page2
+	^HelpTopic 
+		title: 'Enabling Middleware'
+		contents:	
+	'Enabling Middleware
+	
+	globalMiddleware
+
+	You can add global middleware which is user for every handler. Do this, you have to change the class method "globalMiddleware" of your application. It should return an "OrderedCollection" including symbols that match the class names of the middleware you would like to use.
```

Comment:
outdated, please adapt to the changes made in https://github.com/hpi-swa-teaching/RatPack2.0/commit/0bfa2e1a47233ff11e739059d9b4e061bf5644a9#diff-f23f55b0a97668bcb98b28c92027532f.

The global middleware list is stored in the settings object. Initialization can be done in RPApplication's `initializeSettings` method, which is only there for the user to apply his settings.

Reply:
If you know what and how to adapt please do it. Have no time